labonneboite/scripts/create_index.py

```diff
@@ -303,7 +304,13 @@ def get_scores_by_rome_and_boosted_romes(office, office_to_update=None):
     # elasticsearch does not understand sets, so we use a dict of 'key => True' instead
     boosted_romes = {}
     boosted_alternance_romes = {}
-    
+
+    if PSE_STUDY:
+        sirets_to_remove_pse = load_siret_to_remove()
+        office_to_remove_pse = True if office.siret in sirets_to_remove_pse else False
```

Comment:
je pense que ça se simplifie en `office_to_remove_pse = (office.siret in sirets_to_remove_pse)`